src/roslibpy/core.py

```diff
@@ -30,6 +31,59 @@ def __init__(self, values=None):
             self.update(values)
 
 
+class Header(UserDict):
+    """Represents a message header of the ROS type std_msgs/Header."""
+    def __init__(self, seq=None, stamp=None, frame_id=None):
+        self.data = {}
+        self.data['seq'] = seq
+        self.data['stamp'] = Time(stamp['secs'], stamp['nsecs']) if stamp else None
+        self.data['frame_id'] = frame_id
+
+
+class Time(UserDict):
+    """Represents ROS time with two integers: seconds since epoch and nanoseconds since seconds."""
+    def __init__(self, secs, nsecs):
+        self.data = {}
+        self.data['secs'] = secs
+        self.data['nsecs'] = nsecs
+
+    @property
+    def secs(self):
+        """Seconds since epoch."""
+        return self.data['secs']
+
+    @property
+    def nsecs(self):
+        """Nanoseconds since seconds."""
+        return self.data['nsecs']
+
+    def is_zero(self):
+        """Return ``True`` if zero (secs and nsecs) otherwise ``False``."""
+        return self.data['secs'] == 0 and self.data['nsecs'] == 0
+
+    def to_nsec(self):
+        """Return time as nanoseconds from epoch."""
+        stamp_secs = self.data['secs']
+        stamp_nsecs = self.data['nsecs']
+        return stamp_secs * 1000000000 + stamp_nsecs
```

Comment:
I find `1000000000` hard to read and would prefer `1e9`, but then it becomes a float. So, as you prefer.

Reply:
done at https://github.com/gramaziokohler/roslibpy/pull/72/commits/d3f0ab649f4db8b2a6362e07260a6474a0332c08